modules/configuring-hybrid-ovnkubernetes.adoc

```diff
@@ -0,0 +1,85 @@
+//
+// * installing/installing_aws/installing-aws-network-customizations.adoc
+// * installing/installing_azure/installing-azure-network-customizations.adoc
+// * networking/ovn_kubernetes_network_provider/configuring-hybrid-networking.adoc
+
+[id="configuring-hybrid-ovnkubernetes_{context}"]
+= Configuring hybrid networking with OVN-Kubernetes
+
+You can configure your cluster to use hybrid networking with OVN-Kubernetes. This allows a hybrid cluster that supports different node networking configurations. For example, this is necessary to run both Linux and Windows nodes in a cluster.
+
+[IMPORTANT]
+====
+You must configure hybrid networking with OVN-Kubernetes during the installation of your cluster. You cannot switch to hybrid networking after the installation process.
+====
+
+.Prerequisites
+
+* You defined `OVNKubernetes` for the `networking.networkType` parameter in the `install-config.yaml` file.
+
+.Procedure
+
+. Use the following command to create manifests:
++
+[source,terminal]
+----
+$ ./openshift-install create manifests --dir=<installation_directory> <1>
+----
+<1> For `<installation_directory>`, specify the name of the directory that contains the `install-config.yaml` file for your cluster.
+
+. Create a file that is named `cluster-network-03-config.yml` in the `<installation_directory>/manifests/` directory:
++
+[source,terminal]
+----
+$ touch <installation_directory>/manifests/cluster-network-03-config.yml <1>
+----
+<1> For `<installation_directory>`, specify the directory name that contains the `manifests/` directory for your cluster.
++
+After creating the file, several network configuration files are in the `manifests/` directory, as shown:
++
+[source,terminal]
+----
+$ ls <installation_directory>/manifests/cluster-network-*
+----
++
+.Example output
+[source,terminal]
+----
+cluster-network-01-crd.yml
+cluster-network-02-config.yml
+cluster-network-03-config.yml
+----
+
+. Open the `cluster-network-03-config.yml` file and configure OVN-Kubernetes with hybrid networking. For example:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  creationTimestamp: null
+  name: cluster
+spec: <1>
+  clusterNetwork: <2>
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  externalIP:
+    policy: {}
+  serviceNetwork:
+  - 172.30.0.0/16
+  defaultNetwork:
+    type: OVNKubernetes <3>
+    ovnKubernetesConfig:
+      hybridOverlayConfig:
+        hybridClusterNetwork: <4>
+        - cidr: 10.132.0.0/14
+          hostPrefix: 23
+status: {}
+----
+<1> The parameters for the `spec` parameter are only an example. Specify your configuration for the Cluster Network Operator in the custom resource.
+<2> Specify the network configurations used when adding nodes.
+<3> Specify `OVNKubernetes` as the default Container Network Interface (CNI).
+<4> Specify the network configurations given to nodes on the additional overlay network. The `hybridClusterNetwork` CIDR cannot overlap with the `clusterNetwork` CIDR.
```

Comment:
In this case, I think the _only_ items in this list are cluster network CIDRs.

So _Specify the network configurations_ is overly broad.